hawkular-dmr-client/src/main/java/org/hawkular/dmr/api/OperationBuilder.java

```diff
@@ -765,4 +790,9 @@ public T valueAttribute(String value) {
         return new WriteAttributeOperationBuilder<>(null);
     }
 
+    @SuppressWarnings("unchecked")
+    public static GenericOperationBuilder<GenericOperationBuilder<?>> generic(String operationName) {
```

Comment:
Oh! The "generic" irony :).

It defines the operation name, useful when you don't know exactly the operation that is going to be executed until runtime.

I chose "generic" to follow other API calls:

Builder | method
---------  | ----------
AddOperationBuilder | add()
MapPutOperationBuilder | mapPut()
ReadResourceOperationBuilder | readResource()
GenericOperationBuilder | generic()

However, I could change the name and/or add the javadoc to make it more helpful.